cli/wallet/nep17.go

```diff
@@ -111,9 +112,12 @@ func newNEP17Commands() []cli.Command {
 		{
 			Name:      "transfer",
 			Usage:     "transfer NEP17 tokens",
-			UsageText: "transfer --wallet <path> --rpc-endpoint <node> --timeout <time> --from <addr> --to <addr> --token <hash> --amount string",
+			UsageText: "transfer --wallet <path> --rpc-endpoint <node> --timeout <time> --from <addr> --to <addr> --token <hash> --amount string [data]",
```

Comment:
I'm not sure we want to make it positional. We have all the other things as proper named arguments.

Reply:
Data can be of the form `[ data1 data2 ]` (spaces are present). Not sure that named args can have spaces.

Reply:
They can, but you'll have to parse the line which requires additional (errors-prone) code. OK, let's go with positionals, all the other code is ready for this.